### PR TITLE
Wireguard bb update

### PR DESCRIPTION
**Pull request by estimadarocha**

### CODE REVIEW

**Comment by atvcaptain** [comment]
build error

h/siphash.c:451:43: error: 'fallthrough' undeclared (first use in this function)
|   451 |         case 3: b |= ((u32)end[2]) << 16; fallthrough;
|       |                                           ^~~~~~~~~~~

File: meta-oe/recipes-kernel/wireguard/wireguard-module_1.0.20211208.bb [hunk 1]
```diff
@@ -1,5 +1,12 @@
 require wireguard.inc
 
+SRC_URI:append = "file://fallthrough.patch"
```
Comment (catalinii) [inline meta-oe/recipes-kernel/wireguard/wireguard-module_1.0.20211208.bb]: Move this after line 7

**Review by catalinii — APPROVED** [review]

**Comment by catalinii** [comment]
@atvcaptain can you please merge this PR ?

**Comment by catalinii** [comment]
Tested this on gbultraueh and works as expected.

**Comment by estimadarocha** [comment]
> Tested this on gbultraueh and works as expected.

@catalinii thanks for the feedback

**Comment by atvcaptain** [comment]
i check it today

**Comment by atvcaptain** [comment]
rework patch add and update to version from your commit

**Pull request closed without merging**